src/pages/AboutPage.tsx

```diff
@@ -1,9 +1,50 @@
-import React from 'react'
-
 const AboutPage = () => {
-  return (
-    <div>AboutPage</div>
-  )
-}
+  const About =
+    "FilmWise: Your Ultimate Movie Hub. Experience movies like never before with FilmWise, the web application that allows you to dive into detailed movie information, share your thoughts through comments and reviews, and connect with fellow cinephiles. Discover, discuss, and decide on your favorite films, all in one place. FilmWise is the ultimate movie hub for all your cinematic needs. You can search for movies and dramas by name or genre. If you like a movie that is not on the list, you can add it so that your friends can see it.";
+  const Team = [
+    {
+      id: 1,
+      image: "https://avatars.githubusercontent.com/u/35267447?v=4",
+      name: "Abu Rihan",
+      role: "Frontend + Backed",
+      github: "https://github.com/raihan2bd",
+      linkdin: "https://www.linkedin.com/in/raihan2bd/",
+      Twitter: "https://twitter.com/raihan2bd",
+    },
+    {
+      id: 2,
+      image: "https://avatars.githubusercontent.com/u/80938128?v=4",
+      name: "Hassaan Baig",
+      role: "Frontend",
+      github: "https://github.com/hassaanjbaig-code",
+      linkdin: "https://www.linkedin.com/in/hassaan-jawwad-baig/",
+    }
+  ]
+ return <section className="w-full md:h-screen  background_gradient">
```

Comment:
- [ ] [Optional] @Hassaanjbaig-code Your `About page` is looking good. I like the gradient background, However, it can be improved if you add padding on the x-axis it will look better in the mobile version.
![Capture](https://github.com/raihan2bd/filmwise-front/assets/35267447/3a18a824-77fd-4ff8-b34a-13caaa3b42aa)
![Capture2](https://github.com/raihan2bd/filmwise-front/assets/35267447/db23fa09-818e-43bc-8da1-da73975932f9)